web/src/i18n/locales/en.json

```diff
@@ -1,3 +1,117 @@
+        "description":
+          "Sed ut perspiciatis unde omnis iste natus error sit voluptatem accusantium doloremque laudantium, totam rem aperiam, eaque ipsa quae ab illo inventore veritatis et quasi architecto...",
+        "prompt": "Describe how you'll meet the Leverage condition"
+      },
+      "bizResults": {
+        "title": "Business Results",
+        "description":
+          "Sed ut perspiciatis unde omnis iste natus error sit voluptatem accusantium doloremque laudantium, totam rem aperiam, eaque ipsa quae ab illo inventore veritatis et quasi architecto...",
+        "prompt": "Describe how you'll meet the Business Results condition"
+      },
+      "reporting": {
+        "title": "Reporting",
+        "description":
+          "Sed ut perspiciatis unde omnis iste natus error sit voluptatem accusantium doloremque laudantium, totam rem aperiam, eaque ipsa quae ab illo inventore veritatis et quasi architecto...",
+        "prompt": "Describe how you'll meet the Reporting condition"
+      },
+      "interoperability": {
+        "title": "Interoperability",
+        "description":
+          "Sed ut perspiciatis unde omnis iste natus error sit voluptatem accusantium doloremque laudantium, totam rem aperiam, eaque ipsa quae ab illo inventore veritatis et quasi architecto...",
+        "prompt": "Describe how you'll meet the Interoperability condition"
+      },
+      "mitigation": {
+        "title": "Mitigation Strategy",
+        "description":
+          "Sed ut perspiciatis unde omnis iste natus error sit voluptatem accusantium doloremque laudantium, totam rem aperiam, eaque ipsa quae ab illo inventore veritatis et quasi architecto...",
+        "prompt": "Describe how you'll meet the Mitigation Strategy condition"
+      },
+      "keyPersonnel": {
+        "title": "Key Personnel",
+        "description":
+          "Sed ut perspiciatis unde omnis iste natus error sit voluptatem accusantium doloremque laudantium, totam rem aperiam, eaque ipsa quae ab illo inventore veritatis et quasi architecto...",
+        "prompt": "Describe how you'll meet the Key Personnel condition"
+      },
+      "documentation": {
+        "title": "Documentation",
+        "description":
+          "Sed ut perspiciatis unde omnis iste natus error sit voluptatem accusantium doloremque laudantium, totam rem aperiam, eaque ipsa quae ab illo inventore veritatis et quasi architecto...",
+        "prompt": "Describe how you'll meet the Documentation condition"
+      },
+      "minimizeCost": {
+        "title":
+          "Strategies to Minimize Cost and Difficulty on Alternative Hardware or Operating System",
+        "description":
+          "Sed ut perspiciatis unde omnis iste natus error sit voluptatem accusantium doloremque laudantium, totam rem aperiam, eaque ipsa quae ab illo inventore veritatis et quasi architecto...",
+        "prompt": "Describe how you'll meet the Cost Minimization condition"
+      }
+    }
+  }
```

Comment:
Would love thoughts on this nesting structure - should we have that, or should it be flat?  I.e., do we want `activities.standardsAndConditions.minimizeCost` or do we want (e.g.) `act-standards-minimize`?  The former is more complicated but the latter might be harder for humans to interpret.

Reply:
I think you're right on this re: interpretation.